doc/source/cookbook.rst

```diff
@@ -8,20 +8,13 @@
    import pandas as pd
    import numpy as np
    from pandas.compat import StringIO
-
-   import random
    import os
    import itertools
    import functools
    import datetime
-
+   import glob
```

Comment:
We have a lot of concurrent isort work going on - can we reflect that with the import order here as well?

Reply:
@saurav2608 the order should be the one defined in pep8:
1. Python standard library
2. Third party libraries
3. Our own code

And in each of them, order the libraries alphabetically.

Can you update please, so we can merge?